test/specs/Class2/Homework-1.js

```diff
@@ -1,3 +1,60 @@
+
+
+const { expect } = require("chai");
+const { it } = require("mocha");
+
+describe('WebElements Test cases', () => {
+
+    it('Create new account, gender not selected', async () => {
+
+        /**
+         * 1. Launch facebook.com
+         * 2. Click Create New Account button
+         */
+
+        // 1. Launch facebook.com
+        await browser.url('/');
+
+        // 2. Click Create New Account button
+        const createNewAccountButton = await $('=Create New Account');
+        createNewAccountButton.click();
+
+         await browser.pause(7000);
+
+         // 3. Verify female-gender button is not selected
+        const femaleRadioButton = await $('input[value="1"]');
+        const isFemaleGenderSelected = await femaleRadioButton.isSelected();
+        expect(isFemaleGenderSelected, 'Female gender is already selected').to.be.false;
+
+        await browser.pause(5000);
+
+        // 4. Verify male-gender button is not selected
+        const maleRadioButton = await $('[input[value="2"]');
+        const isMaleGenderSelected = await maleRadioButton.isSelected();
+        expect(isMaleGenderSelected, 'Male gender is already selected').to.be.false;
+
+        await browser.pause(5000);
+
+        // 5. Verify custom-gender button is not selected
+        const customRadioButton= await$('[input[value="-1"]');
+        const isCustomGenderSelected = await customRadioButton.isSelected();
+        expect (isCustomGenderSelected, 'Custom gender is already selected').to.be.false;
+
+        await browser.pause(5000);
+ 
+        // 6. If female gender is NOT selected, then click on female gender radio button
+        const femaleRadioButtonClick = await $('button id=u_13_4_gr');
```

Comment:
Pls add if-loop , bcz requirement says "If female gender is NOT selected"

Reply:
@deepaktechnosoft updated